Moving docs from beta to v1.0 for Terms of Use API's (#11642)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/agreementacceptance.md

---
title: "agreementAcceptance resource type"
description: "Represents the current status of a user within scope of a company's customizable terms of use powered by Azure Active Directory (Azure AD)."
localization_priority: Normal
doc_type: resourcePageType

> ms.prod: "microsoft-identity-platform"

Reviewer: ms.prod should be governance.

author: raprakasMSFT
---

# agreementAcceptance resource type

Namespace: microsoft.graph

Represents the current status of a user within scope of a company's customizable terms of use powered by Azure Active Directory (Azure AD).

<!--
## Methods

| Method       | Return Type | Description |
|:-------------|:------------|:------------|
| [Get agreementAcceptance](../api/agreementacceptance-get.md) | [agreementAcceptance](agreementacceptance.md) | Read properties and relationships of agreementAcceptance object. |
| [Update](../api/agreementacceptance-update.md) | [agreementAcceptance](agreementacceptance.md) | Update an **agreementAcceptance** object. |
| [Delete](../api/agreementacceptance-delete.md) | None | Delete an **agreementAcceptance** object. |
-->

## Properties
| Property     | Type        | Description |
|:-------------|:------------|:------------|

> |agreementFileId|String|ID of the agreement file accepted by the user.|

Reviewer: ```suggestion
|agreementFileId|String|The identifier of the agreement file accepted by the user.|
```

> |agreementId|String|ID of the agreement.|

Reviewer: ```suggestion
|agreementId|String|The identifier of the agreement.|
```

> |deviceDisplayName|String|The display name of the device used for accepting the agreement.|

Reviewer: ```suggestion
|deviceDisplayName|String|The display name of the device used to accept the agreement.|
```

|deviceId|String|The unique identifier of the device used for accepting the agreement.|

> |deviceOSType|String|The operating system used for accepting the agreement.|

Reviewer: ```suggestion
|deviceOSType|String|The operating system used to accept the agreement.|
```

> |deviceOSVersion|String|The operating system version of the device used for accepting the agreement.	|

Reviewer: ```suggestion
|deviceOSVersion|String|The operating system version of the device used to accept the agreement.	|
```

|expirationDateTime|DateTimeOffset|The expiration date time of the acceptance. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`|

> |id|String| Read-only.|

Reviewer: ```suggestion
|id|String| The identifier of the agreement acceptance. Read-only.|
```

|recordedDateTime|DateTimeOffset|The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`|

> |state|string| Possible values are: `accepted`, `declined`.|

Reviewer: ```suggestion
|state|string| The state of the agreement acceptance. Possible values are: `accepted`, `declined`.|
```

|userDisplayName|String|Display name of the user when the acceptance was recorded.|
|userEmail|String|Email of the user when the acceptance was recorded.|

> |userId|String|ID of the user who accepted the agreement.|

Reviewer: ```suggestion
|userId|String|The identifier of the user who accepted the agreement.|
```
